specification/communication/data-plane/CallingServer/preview/2021-06-15-preview/communicationservicescallingserver.json

```diff
@@ -0,0 +1,1467 @@
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal server error.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/calling/serverCalls/{serverCallId}/:playAudio": {
+      "post": {
+        "tags": [
+          "ServerCall"
+        ],
+        "summary": "Play audio in a call.",
+        "description": "Play audio in a call.",
+        "operationId": "ServerCall_PlayAudio",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "serverCallId",
+            "in": "path",
+            "description": "The server call id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "request",
+            "in": "body",
+            "description": "Play audio request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PlayAudioRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Returns the play audio response.",
+            "schema": {
+              "$ref": "#/definitions/PlayAudioResult"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal server error.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CallModality": {
+      "enum": [
+        "audio",
+        "video"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CallModality",
+        "modelAsString": true
+      }
+    },
+    "EventSubscriptionType": {
+      "enum": [
+        "participantsUpdated",
+        "dtmfReceived"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "EventSubscriptionType",
+        "modelAsString": true
+      }
+    },
+    "CreateCallRequest": {
+      "description": "The request payload for create call.",
+      "required": [
+        "targets",
+        "source",
+        "callbackUri"
```

Comment:
How is the callbackUri called/secured? See the [subscriptions](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#15-push-notifications-via-webhooks) section of the API guidelines to learn more about the expected call sequence...

Reply:
How was the client secret provided? Is the client expected to include the query parameter in the callbackUri when setting up the subscription (e.g. when creating the callConnection)? Or is a property of the ACS resource itself?

Reply:
the client should include it in the callbackUri when creating the subscription.